### ISSUE TITLE
[BUG] Fuel Capacity vs Usable fuel

### ISSUE THREAD

**Issue by Cugix** [flybywiresim/a32nx#2545]
**Mod Version**
Master

**Describe the bug**
In issue 2285 the fuel capacity was reported to be incorrect referencing documents stating "usable fuel". While the usable fuel on each tank is stated correctly the actual capacity of the tanks combined should add up to the 7060 USG as stated on the Airbus websites. Failing to include these 800 USG of fuel should lead to a weight descrepancy and thus a performance difference with the real aicraft. Capacity is what the tank can hold, usuable fuel is the amount that's actually able to be used in that tank.

**To Reproduce**

1. Compare data from the linked issue with Airbus website

**Expected behavior**
Total Fuel tank capacity of 7060 USG of which 6267 USG usable

**Actual behavior**
Fuel tank capacity of 7060 USG

**References**
https://github.com/flybywiresim/a32nx/issues/2285#issue-759074631
https://www.airbus.com/aircraft/passenger-aircraft/a320-family/a320neo.html

**Additional context**
NIL

Was this working before/when did the issue start occurring? 
I don't know if the vanilla version lets you use up all 7060 USG with the latest Master the fuel capacity is wrong.

Is this a problem in the vanilla unmodded game?
Both Vanilla and with FBW A20N

Discord username (if different from GitHub): Cugix#3435

**Comment by donstim** [comment]
This is an MSFS bug. After testing, I found that MSFS allows you to use all of the fuel in all tanks, even when you specify an unusable fuel quantity for each tank. Because of this,, the fuel tank capacities must be set to the usable fuel quantity even though it does cause the airplane weight to be off by a miniscule amount. The total unusable fuel quantity for all tanks is only 21.6 USG (82.1 liters), which is only about 145 lb (65.6 kg).

Including the unusable fuel, fuel tank capacity for the A320neo being modeled is 6288.6 USG, not 7060 USG. This is from official documents, shown in #2286 , not from some webpage.

**Comment by Cugix** [comment]
I do not agree this method applied is a fix. You found a bug in MSFS and created an inconsistency even with your own sources to work around that bug. I feel there's still a reason for the Airbus stating 7060 USG as the airplanes capacity, it's an oddly specific number for it to be just made up. I don't have access to the FCOM or AFM but there might be more places where unusable fuel exists in the fuel system.

**Comment by donstim** [comment]
We did not create an inconsistency. We matched the actual fuel tank capacities to the specific airplane variant we are modeling. We used official sources to document that capacity and to verify it is correct. We are not "working around" the MSFS bug. It still exists, but in this case it is very inconsequential. If you feel strongly about it, simply add 145 lb to either your payload or your empty weight.

The figure you are quoting from an Airbus website is marketing material that states a capability that can be purchased. There is a capability to fit 2 additional center fuel tanks to the airplane as in previous versions of the A320. However, no one has yet purchased an option for even 1 additional center tank, so those options have not even been certified yet.
